src/main/scala/org/broadinstitute/dsde/workbench/sam/schema/JndiSchemaDAO.scala

```diff
@@ -78,9 +81,34 @@ class JndiSchemaDAO(protected val directoryConfig: DirectoryConfig)(implicit exe
       _ <- removePolicySchema()
       _ <- removeWorkbenchGroupSchema()
       _ <- removeWorkbenchPetServiceAccountSchema()
+      _ <- removeWorkbenchPersonSchema()
     } yield ()
   }
 
+  private def createWorkbenchPersonSchema(): Future[Unit] = withContext { ctx =>
+    val schema = ctx.getSchema("")
+
+    createAttributeDefinition(schema, "1.3.6.1.4.1.18060.0.4.3.2.30", Attr.proxyEmail, "proxy group email", true)
```

Comment:
The IDs here are unique within sam and uses the same base but the final number was chosen arbitrarily. This is all mainly copying similar examples, primarily for pet service accounts. There's a good chance that there will be _something_ that needs tweaking here.

Reply:
if they are unique within sam then 👍